doc/source/tutorial/stats/rv_infrastructure.md

```diff
@@ -0,0 +1,920 @@
+.. jupyterlite:: ../../_contents/hypothesis_bartlett.ipynb
+   :new_tab: True
+```
+
+# Random Variable Transition Guide
+
++++
+
+## Background
+
++++
+
+Prior to SciPy 1.15, all of SciPy's continuous probability distributions (e.g. `scipy.stats.norm`) have been instances of subclasses of `scipy.stats.rv_continuous`.
+
+```{code-cell} ipython3
+from scipy import stats
+dist = stats.norm
+type(dist)
+```
+
+```{code-cell} ipython3
+isinstance(dist, stats.rv_continuous)
+```
+
+There were two obvious ways to work these objects.
+
+According to the more common way, both "arguments" (e.g. `x`) and "distribution parameters" (e.g. `loc`, `scale`) were provided as inputs to methods of the object.
+
+```{code-cell} ipython3
+x, loc, scale = 1., 0., 1.
+dist.pdf(x, loc, scale)
+```
+
+The less common approach was to invoke the `__call__` method of the distribution object, which returned an instance of `rv_continuous_frozen`, regardless of the original class.
+
+```{code-cell} ipython3
+frozen = stats.norm()
+type(frozen)
+```
+
+Methods of this new object accept only arguments, not the distribution parameters.
+
+```{code-cell} ipython3
+frozen.pdf(x)
+```
+
+In a sense, the instances of `rv_continuous` like `norm` represented "distribution families", which require parameters to identify a particular probability distribution, and an instance of `rv_continuous_frozen` was akin to a "random variable" - a mathematical object that follows a particular probability distribution.
+
+Both approaches are valid and have advantages in certain situations. For instance, `stats.norm.pdf(x)` appears more natural than `stats.norm().pdf(x)` for such a simple invocation or when methods are viewed as functions of distribution parameters rather than the usual arguments (e.g., likelihood). However, the former approach has a few inherent disadvantages; e.g., all of SciPy's 125 continuous distributions have to be instantiated at import time, distribution parameters must be validated every time a method is called, and documentation of methods must either a) be generated separately for every method of every distribution or b) omit the shape parameters that are unique for each distribution.
+]
+To address these and other shortcomings, [gh-15928](https://github.com/scipy/scipy/issues/15928) proposed a new, separate infrastructure based on the latter (random variable) approach. This transition guide documents how users of `rv_continuous` and `rv_continuous_frozen` can migrate to the new infrastructure.
+
+*Note: The new infrastructure may not yet be as convenient for some use cases, especially fitting distribution parameters to data. Users are welcome to continue to use the old infrastructure if it suits their needs; it is not deprecated at this time.*
+
++++
+
+## Basics
+
++++
+
+In the new infrastructure, distributions families are classes named according to `CamelCase` conventions. They must be instantiated before use, with parameters passed as keyword-only arguments.
```

Comment:
```suggestion
In the new infrastructure, distributions families are classes named according to `CamelCase` conventions (note that not all distributions have a corresponding class yet; see `make_distribution` below). They must be instantiated before use, with parameters passed as keyword-only arguments.
```

Reply:
👍 I put something like this a little here.